rust/protocol/src/sealed_sender.rs

```diff
@@ -1001,12 +1353,22 @@ pub async fn sealed_sender_multi_recipient_encrypt<R: Rng + CryptoRng>(
 
     serialized.extend_from_slice(e_pub.public_key_bytes()?);
     serialized.extend_from_slice(&ciphertext);
-    serialized.extend_from_slice(&tag);
 
     Ok(serialized)
 }
 
-// For testing
+/// Split out the encoded message from [sealed_sender_multi_recipient_encrypt] into a sequence of
+/// individual encrypted [UnidentifiedSenderMessageContent]s. **Note: this method is only used in
+/// testing.**
+///
+/// This method strips recipients' metadata and splits a bulk v2 sealed-sender message into byte
+/// strings which can be processed by [sealed_sender_decrypt_to_usmc]. This is used in our
+/// Java/Swift/TypeScript tests. For the Signal app, this process of splitting out a v2
```

Comment:
I'd strike "this is used in our tests". It's used in our Rust tests too! It's free for clients to use! The note above gets the point across already.

Reply:
This makes sense! The point about stripping the metadata needed to route the message was I think all I wanted to make clear in my original attempt at this documentation change, so I have removed this sentence!